src/pseudo/PspUpf.jl

```diff
@@ -128,12 +121,33 @@ function PspUpf(path; identifier=path)
         ρcore = zeros(Float64, length(rgrid))
     end
 
+    #* HACK!
+    if !isnothing(rcut)
```

Comment:
Not really a hack. I think removing the rcut is fine.

Also at least put one test somewhere to ensure the code is run, so that we don't accidentially break it.